.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,59 @@
+name: Bug Report
+description: Report something that's not working correctly
+labels: kind/bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Please fill the form below.  
```

Comment:
id recommend on avoiding the word please in this template
most ux writing standard recommend leading with the action you need to take, and please can distract from this. if its important for you to keep it, id keep it here, but then drop it in the rest of the questions.

Reply:
Dropped.